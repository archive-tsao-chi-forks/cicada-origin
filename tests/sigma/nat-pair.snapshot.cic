def one = add1(zero)
def two = add1(one)
def three = add1(two)

def one_two: (Nat) * Nat = cons(one, two)

cdr(one_two)
