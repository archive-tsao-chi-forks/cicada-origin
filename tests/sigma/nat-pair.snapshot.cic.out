2: Nat 
3: Nat 
4: Nat 

